pages/task/read/views/models/pil.jsx

```diff
@@ -125,44 +130,52 @@ export default function PIL({ task, values }) {
       }
     </StickyNavAnchor>,
 
-    <StickyNavAnchor id="training" key="training">
-      <h2><Snippet>sticky-nav.training</Snippet></h2>
-      {
-        profile.certificates && profile.certificates.length > 0
-          ? <Modules certificates={profile.certificates} />
-          : <p><em><Snippet>pil.training.none</Snippet></em></p>
-      }
-    </StickyNavAnchor>,
+    (
+      (!isReview || profile.certificates.length > 0) && (
+        <StickyNavAnchor id="training" key="training">
+          <h2><Snippet>sticky-nav.training</Snippet></h2>
+          {
+            profile.certificates && profile.certificates.length > 0
+              ? <Modules certificates={profile.certificates} />
+              : <p><em><Snippet>pil.training.none</Snippet></em></p>
+          }
+        </StickyNavAnchor>
+      )
+    ),
 
-    <StickyNavAnchor id="exemptions" key="exemptions">
-      <h2><Snippet>sticky-nav.exemptions</Snippet></h2>
-      {
-        profile.exemptions && profile.exemptions.length > 0
-          ? profile.exemptions.map((exemption, index) => (
-            <div key={index}>
-              <dl>
-                <dt><Snippet>pil.exemptions.module</Snippet><span>:</span></dt>
-                <dd>{exemption.module}
-                  {
-                    exemption.species && exemption.species.length > 0 &&
-                      (
-                        <Fragment>
-                          { exemption.species.map((s, index) => (
-                            <p key={index}>{s}</p>
-                          ))}
-                        </Fragment>
-                      )
+    (
+      (!isReview || profile.certificates.length > 0) && (
```

Comment:
Can we consolidate this condition into a `const showTraining = !isReview || profile.certificates.length > 0;` and use that in the various place have this line?